lib/rust/ensogl/core/src/data/color/space/def.rs

```diff
@@ -389,6 +389,60 @@ impl Rgb {
         Self::new(r.into() / 255.0, g.into() / 255.0, b.into() / 255.0)
     }
 
+    /// Return a color if `css_hex` is a `#RGB` or `#RRGGBB` string, where `R`, `G`, `B` represent
+    /// lower- or upper-case hexadecimal digits. The RR, GG, BB color componets are mapped from [00
+    /// - ff] value range into [0.0 - 1.0] (a three-digit string `#RGB` is equivalent to a
+    /// six-digit string `#RRGGBB` constructed by duplicating the digits).
+    ///
+    /// The format is adapted from the hexadecimal color notation used in CSS - see:
+    /// https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color
+    /// ```
+    /// # use assert_approx_eq::assert_approx_eq;
+    /// # use ensogl_core::data::color::Rgb;
+    /// # const PRECISION_8_BIT: f32 = 0.001;
+    ///
```

Comment:
what is this syntax?

Reply:
As discussed, this is a rustdoc syntax that allows hiding some lines of code in runnable examples - https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#hiding-portions-of-the-example.

Reply:
I didn't know that! Thanks!